documentation/asciidoc/topics/proc_installing_openshift_console.adoc

```diff
@@ -0,0 +1,35 @@
+[id='installing-openshift-console_{context}']
+= Installing the Helm chart through the {openshiftshort} console
+
+[role="_abstract"]
+The {brandname} Helm chart is available from the Red Hat developer catalog so you can install it directly from the {osweb}.
+During the installation process, you can pass values to the chart to specify configuration for your {brandname} cluster.
+
+.Prerequisites
+
+* Have access to an {openshiftshort} cluster.
+
+.Procedure
+
+. Log in to the {osweb}.
+. Select the **Developer** perspective.
+. Select the **Helm** category and navigate to the Red Hat developer catalog.
```

Comment:
This isn't quite right. The steps are:

. Select the **Developer** perspective.
. Navigate to the **+ Add** screen
. Click **Helm Chart**
...

Reply:
Right, I see the steps now. I did get there on OCD via the steps I took so maybe we should make it a little more generic in case it changes. I kind of want to be careful about documenting the OS UI too much.